File: app/src/main/java/com/kakacat/minitool/common/BaseActivity.java
package com.kakacat.minitool.common;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.facebook.drawee.backends.pipeline.Fresco;

public class BaseActivity extends AppCompatActivity{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Fresco.initialize(this);
    }

}

File: app/src/main/java/com/kakacat/minitool/common/base/BaseActivity.kt
package com.kakacat.minitool.common.base


import android.content.Context
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity

open class BaseActivity : AppCompatActivity(), IContext {

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
            return true
        }
        return false
    }

    override val context: Context
        get() = this
}